src/components/eyewitness/eyewitness.tsx

```diff
@@ -1,96 +1,136 @@
-import { Button, LoadingOverlay, Stack, TextInput, Title } from "@mantine/core";
+import { Button, Stack, TextInput, Title } from "@mantine/core";
```

Comment:
Removed because not needed.
```suggestion
import { Button, Stack, TextInput } from "@mantine/core";
```